packages/vue-components/stories/Welcome.mdx

```diff
@@ -21,12 +20,11 @@ regarding their props or events.
 This components library use PrimeVue + TailwindCSS.
 
 Several semantic colors are available :
-* primary (based on the blue of logo)
-* secondary (based on the orange of logo)
+
+- primary (based on the blue of logo)
+- secondary (based on the orange of logo)
 
 <div class="bg-primary p-4 m-4 text-2xl flex items-center justify-center text-white">
   Primary color
 </div>
-<div class="bg-secondary p-4 m-4 text-2xl flex items-center justify-center">
-  Secondary color
-</div>
\ No newline at end of file
+<div class="bg-secondary p-4 m-4 text-2xl flex items-center justify-center">Secondary color</div>
```

Comment:
I imagine that it is the linter which performed this change.

If I'm right, could we disable this kind of correction?

That brings more noise than real value in my opinion.

Reply:
Mmm, I don't want to remove this changes, as it allows us to have a consistent base of code.
Nevertheless, in this case, that's my fault as I created this `.mdx` file without applying the linter.
I think that kind of changes won't occur often.